src/panoramic/cli/schemas/config.schema.json

```diff
@@ -65,16 +75,50 @@
           },
           "allOf": [
             {
-              "if": {"properties": {"type": { "const": "postgres" }}},
-              "then": {"required":  ["database", "host"]}
+              "if": {
+                "properties": {
+                  "type": {
+                    "const": "postgres"
+                  }
+                }
+              },
+              "then": {
+                "required": [
+                  "database",
+                  "host"
+                ]
+              }
             },
             {
-              "if": {"properties": {"type": { "const": "snowflake" }}},
-              "then": {"required":  ["account"]}
+              "if": {
+                "properties": {
+                  "type": {
+                    "const": "snowflake"
+                  }
+                }
+              },
+              "then": {
+                "required": [
+                  "account",
+                  "role",
+                  "warehouse"
```

Comment:
@vetyy added these two to match what's in the web UI dialog when the user connects the WH - will allow for easier syncing from CLI to API one day

Reply:
👍